src/dosbox.cpp

```diff
@@ -821,12 +821,16 @@ void DOSBOX_Init()
 	pint->Set_help("The OPL waveform is now sampled at the mixer's playback rate to avoid\n"
 	               "resampling.");
 
+	pbool = secprop->Add_bool("cms", when_idle, true);
+	pbool->Set_help("Enable or disable the CMS for SoundBlaster 1, 1.5 and 2.");
```

Comment:
For consistency with the other settings, let's go with this (e.g., look up `gus`, `imfc`, `ps1audio`, etc.).

`Enable CMS emulation for SoundBlaster 1, 1.5 and 2 (enabled by default).`